packages/apify/src/utils.ts

```diff
@@ -277,7 +281,7 @@ export async function getMemoryInfo(): Promise<MemoryInfo> {
         const accessPromised = util.promisify(fs.access);
 
         // Check wheter cgroups V1 or V2 is used
-        let cgroupsVersion = 'V1';
+        let cgroupsVersion: keyof typeof MEMORY_FILE_PATHS.TOTAL = 'V1';
```

Comment:
`'V1' as const` should work here too i guess

Reply:
it wouldn't, because it would make the type of `cgroupsVersion` to `V1` only. This also ensures we cannot set it to anything but the keys of that object 👍